ui/raidboss/data/05-shb/dungeon/paglthan.js

```diff
@@ -0,0 +1,102 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+
+export default {
+  zoneId: ZoneId.Paglthan,
+  timelineFile: 'paglthan.txt',
+  timelineTriggers: [
+    {
+      // This is a rear cone attack that always follows Wide Blaster.
+      // It has a cast time of under a GCD, so we pre-warn during Wide Blaster.
+      // Only the sides are safe to call at this moment.
+      id: 'Paglthan Spike Flail',
+      regex: /Spike Flail/,
+      beforeSeconds: 4,
+      response: Responses.goSides(),
+    },
+  ],
+  triggers: [
+    {
+      id: 'Paglthan Critical Rip',
+      netRegex: NetRegexes.startsUsing({ id: '5C4E', source: 'Amhuluk' }),
+      condition: Conditions.caresAboutPhysical(),
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Paglthan Electric Burst',
+      netRegex: NetRegexes.startsUsing({ id: '5C4D', source: 'Amhuluk', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Paglthan Lightning Rod Gain',
+      netRegex: NetRegexes.gainsEffect({ effectId: 'A0E' }),
+      condition: Conditions.targetIsYou(),
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Drop lightning outside',
+        },
+      },
+    },
+    {
+      id: 'Paglthan Lightning Rod Lose',
+      netRegex: NetRegexes.losesEffect({ effectId: 'A0E' }),
+      condition: Conditions.targetIsYou(),
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Away from lightning circle',
+        },
+      },
+    },
+    {
+      id: 'Paglthan Defensive Reaction',
+      netRegex: NetRegexes.startsUsing({ id: '5C9E', source: 'Magitek Core', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Paglthan Twisted Scream',
+      netRegex: NetRegexes.startsUsing({ id: '5B47', source: 'Lunar Bahamut', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Paglthan Akh Morn',
+      netRegex: NetRegexes.headMarker({ id: '005D' }),
+      response: Responses.stackMarkerOn(),
+    },
+    {
+      id: 'Paglthan Mega Flare',
+      netRegex: NetRegexes.headMarker({ id: 'E000' }),
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'Paglthan Kan Rhai',
```

Comment:
This might need a tiny bit more complexity around it, since the move will continue hitting in the place that it's dropped.
In other words, this mechanic is dissimilar from the plus mechanics in Lakshmi (a one-time hit from the player). I don't know if that's obvious or if there should be a MOVE! alert to follow this if it picked you.

Feel free to disagree!

Reply:
I kind of think that's covered by "Drop AoE Outside"? It seems to me that's the typical phrasing we used for shedding persistent environmental hazards, as opposed to targeted Chariot-type AoEs that would be covered by `spread` or `getOut`. I suppose we could do "Drop Cross Puddle Outside"?

Reply:
Oh, I was thinking more along the lines of like a delaySeconds: 3 that shouts `Move!` on the person with the headmarker but that might be excessive.

Reply:
Yeah, I was having the same thought as Panic, in that the surprising part of this mechanic is just the "move away" part at the end, and the "don't hit people with the giant cross on you" is more obvious.